projects/packages/jetpack-mu-wpcom/src/features/custom-css/csstidy/class.csstidy-optimise.php

```diff
@@ -632,11 +632,11 @@ public static function explode_ws( $sep, $string ) {
 	 * @see dissolve_4value_shorthands()
 	 */
 	public static function merge_4value_shorthands( $array ) {
-		$return     = $array;
+		$return     = is_array( $array ) ? $array : array();
 		$shorthands = & $GLOBALS['csstidy']['shorthands'];
 
 		foreach ( $shorthands as $key => $value ) {
-			if ( isset( $array[ $value[0] ] ) && isset( $array[ $value[1] ] )
+			if ( is_array( $value ) && isset( $array[ $value[0] ] ) && isset( $array[ $value[1] ] )
 							&& isset( $array[ $value[2] ] ) && isset( $array[ $value[3] ] ) && $value !== 0 ) {
```

Comment:
Seems just moving the `$value !==0` check at the beginning might work due to the value in [data.inc.php](https://github.com/Automattic/jetpack/blob/trunk/projects/plugins/jetpack/modules/custom-css/csstidy/data.inc.php#L338) file of csstidy.

A similar thing was done [here](https://github.com/Cerdic/CSSTidy/commit/f75bbf97a9320b5b76b65a52b7962ded56e380ab) although the reason is not explicitly stated.

Reply:
Thanks for the suggestion @darssen! Let's try that.